openwisp_utils/admin_theme/templates/admin/edit_inline/help_text_stacked.html

```diff
@@ -0,0 +1,47 @@
+{% comment %}
+Copied from https://github.com/django/django/blob/HEAD/django/contrib/admin/templates/admin/edit_inline/stacked.html
+to add customization.
+{% endcomment %}
+{% load i18n admin_urls %}
+<div class="js-inline-admin-formset inline-group"
+     id="{{ inline_admin_formset.formset.prefix }}-group"
+     data-inline-type="stacked"
+     data-inline-formset="{{ inline_admin_formset.inline_formset_data }}">
+
+<fieldset class="module {{ inline_admin_formset.classes }}">
+  {% if inline_admin_formset.formset.max_num == 1 %}
+    <h2>{{ inline_admin_formset.opts.verbose_name|capfirst }}</h2>
+  {% else %}
+    <h2>{{ inline_admin_formset.opts.verbose_name_plural|capfirst }}</h2>
+  {% endif %}
+{{ inline_admin_formset.formset.management_form }}
+{{ inline_admin_formset.formset.non_form_errors }}
+
+{% for inline_admin_form in inline_admin_formset %}<div class="inline-related{% if inline_admin_form.original or inline_admin_form.show_url %} has_original{% endif %}{% if forloop.last and inline_admin_formset.has_add_permission %} empty-form last-related{% endif %}" id="{{ inline_admin_formset.formset.prefix }}-{% if not forloop.last %}{{ forloop.counter0 }}{% else %}empty{% endif %}">
+  <h3><b>{{ inline_admin_formset.opts.verbose_name|capfirst }}:</b> <span class="inline_label">{% if inline_admin_form.original %}{{ inline_admin_form.original }}{% if inline_admin_form.model_admin.show_change_link and inline_admin_form.model_admin.has_registered_model %} <a href="{% url inline_admin_form.model_admin.opts|admin_urlname:'change' inline_admin_form.original.pk|admin_urlquote %}" class="{% if inline_admin_formset.has_change_permission %}inlinechangelink{% else %}inlineviewlink{% endif %}">{% if inline_admin_formset.has_change_permission %}{% trans "Change" %}{% else %}{% trans "View" %}{% endif %}</a>{% endif %}
+{% else %}#{{ forloop.counter }}{% endif %}</span>
+      {% if inline_admin_form.show_url %}<a href="{{ inline_admin_form.absolute_url }}">{% trans "View on site" %}</a>{% endif %}
+    {% if inline_admin_formset.formset.can_delete and inline_admin_formset.has_delete_permission and inline_admin_form.original %}<span class="delete">{{ inline_admin_form.deletion_field.field }} {{ inline_admin_form.deletion_field.label_tag }}</span>{% endif %}
+  </h3>
+  {% comment %} OpenWISP customization begin {% endcomment %}
+  {% if inline_admin_formset.formset.help_text %}
+    {% with inline_admin_formset.formset.help_text as help_text %}
+        <div class="ow-help-text">
+            <img src="{{ help_text.image_url|default:'/static/admin/img/icon-alert.svg' }}">
+            <p>{{ help_text.text}}
+            {% if help_text.documentation_url %}
+                {% trans 'Please refer to the' %} <a href="{{help_text.documentation_url}}">{% trans 'documentation' %}</a> {% trans 'for more information' %}</p>
```

Comment:
There's a problem here, we cannot split the phrase in 3 different translations, because the order of words changes from language to language.

I think we should use a single [blocktranslate tag](https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#blocktranslate-template-tag) for this and pass the HTML tags as variables (eg: `open_link`, `close_link`).